scripts/update_docker_images.py

```diff
@@ -66,10 +66,11 @@ def update() -> int:
         with open(dockerfile_path, "w") as f:
             f.write(dockerfile_content)
 
-        exit_code = 0
+        update_exists = True
 
-    return exit_code
+    return update_exists
 
 
 if __name__ == "__main__":
-    exit(update())
+    print("NEEDS_UPDATE={}".format(update()))
```

Comment:
nit: instead of printing this, you can probably write directly to `GITHUB_OUTPUT` from within this script: https://github.com/aptos-labs/aptos-core/blob/main/testsuite/determinator.py#L91-L98

Reply:
I left this as-is (and fixed the quote in the workflow) because this way you can see directly in the workflow that it evaluates the output of the python script without needing to open it.